### ISSUE TITLE
TST unit tests fail str comparisons with py3.9

### ISSUE THREAD

**Issue by prjemian** [prjemian/spec2nexus#222]
Unit tests fail with python 3.9 due to the way that strings, str, and byte strings are handled.

* https://github.com/prjemian/spec2nexus/pull/221#issuecomment-724536989
* https://github.com/prjemian/spec2nexus/pull/221#issuecomment-724538604

tests like this fail in py3.9:

```
>           self.assertEqual(value, "")
E           AssertionError: b'' != ''
```

The unit tests need to learn when to expect bytes, str, or string.  See:

* https://medium.com/better-programming/strings-unicode-and-bytes-in-python-3-everything-you-always-wanted-to-know-27dc02ff2686
* https://duckduckgo.com/?t=ffsb&q=python+compare+bytes+with+str&ia=web.

**Comment by prjemian** [comment]
Work on this following #230

**Comment by prjemian** [comment]
Fixed with #233
